### PR TITLE
HTCONDOR-816 send updates timeout

### PR DESCRIPTION
**Pull request by GregThain**

### CODE REVIEW

File: src/condor_negotiator.V6/matchmaker.cpp [hunk 1]
```diff
@@ -1907,7 +1907,7 @@ Matchmaker::forwardAccountingData(std::set<std::string> &names) {
 				// will be zero.  Don't include those submitters.
 
 				if (updateAd.LookupInteger("ResourcesUsed", resUsed)) {
-					cl->sendUpdates(UPDATE_ACCOUNTING_AD, &updateAd, NULL, false);
+					daemonCore->sendUpdates(UPDATE_ACCOUNTING_AD, &updateAd, NULL, false);
```
Comment (JaimeFrey) [inline src/condor_negotiator.V6/matchmaker.cpp:1910]: If we're reducing the use of cl here, let's get rid of it entirely. The two other uses in this function can just call daemonCore->getCollectorList() directly.

**Review by JaimeFrey — APPROVED** [review]
Additional code cleanup in the negotiator would be nice, but not required.
Looks good, otherwise.

File: src/condor_negotiator.V6/matchmaker.cpp (hunk 1 above)
Comment (JaimeFrey) [reply src/condor_negotiator.V6/matchmaker.cpp:1908]: The commit addressing this looks good.

**Review by JaimeFrey — CHANGES_REQUESTED** [review]
The version history entry is under 9.4.0, which has now been branched. The entry needs to be moved to the 9.5.0 list or the branch needs to be merged into V9_4_0-branch (with release manager's permission).

**Comment by timtheisen** [comment]
We will merge this to 9.4.0.

**Pull request merged**